app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt

```diff
@@ -178,31 +178,6 @@ class BrowserToolbarCFRPresenterTest {
         assertNotNull(presenter.tcpCfrPopup)
     }
 
-    @Test
```

Comment:
```
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (195, 37): Cannot access 'text': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (196, 37): Cannot access 'anchor': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (197, 35): Cannot access 'DEFAULT_WIDTH': it is internal in 'Companion'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (197, 56): Cannot access 'properties': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (198, 63): Cannot access 'properties': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (199, 27): Cannot access 'properties': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (200, 27): Cannot access 'properties': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (201, 28): Cannot access 'properties': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (202, 35): Cannot access 'DEFAULT_INDICATOR_START_OFFSET': it is internal in 'Companion'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (202, 73): Cannot access 'properties': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (228, 33): Cannot access 'onDismiss': it is internal in 'CFRPopup'
e: /home/runner/work/fenix/fenix/app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenterTest.kt: (232, 33): Cannot access 'onDismiss': it is internal in 'CFRPopup'
```
Removed these tests because we can't access the properties.